README.md

```diff
@@ -199,7 +199,9 @@ go test ./...
 
 - The `parallel` flag can be used to limit CPU usage, for running tests in the background (`-parallel=4`) - the default is `GOMAXPROCS`
 - The `p` flag can be used to limit the number of _packages_ tested concurrently, if they are interferring with one another (`-p=1`)
-- The `-short` flag skips tests which depend on the database, for quickly spot checking simpler tests in around one minute
+- The `-short` flag skips tests which depend on the database, for quickly spot checking simpler tests in around one minute. We have some validation tests that still need a database url to verify configuration, although they don't actually use the database. You should pass in a fake url to run short tests. `CL_DATABASE_URL='postgresql://doesnotexist:justtopassvalidation@localhost:5432/chainlink_na_test?sslmode=disable' go test -short ./...`
```

Comment:
Which tests still use the DB?

Reply:
It's confusing. The tests don't need a DB, they need a DB url. It's tests like `core/chain/evm/config` which check the DB url to see if it has sufficient password complexity. They fail without the env var set.

Reply:
Hmm that sounds odd. It might just be appropriate to skip them